exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -254,20 +260,22 @@ func (prwe *prwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequ
 			return err
 		}
 		defer resp.Body.Close()
-
 		// 2xx status code is considered a success
 		// 5xx errors are recoverable and the exporter should retry
 		// Reference for different behavior according to status code:
 		// https://github.com/prometheus/prometheus/pull/2552/files#diff-ae8db9d16d8057358e49d694522e7186
 		if resp.StatusCode >= 200 && resp.StatusCode < 300 {
+			prwe.settings.Logger.Debug(fmt.Sprintf("Successfully sent metrics to %s", prwe.clientSettings.Endpoint))
```

Comment:
as above

Reply:
I'm not sure how this log line will benefit troubleshooting. The downside is that this would be very noisy, even at the debug level. This seems more like something that should be captured by a metric produced by the exporter.